app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+class CustomError(Exception):
+    pass
+
+
+class NameCommandError(CustomError):
+    """Command name is not cp"""
+
+
+class CommandError(CustomError):
+    """The error of missing three elements in the team"""
+
+
+def copy_file(command: str) -> None:
+    if command[:2] != "cp":
```

Comment:
why `command[:2] != "cp"` ????
what is meant here?

Reply:
the idea is good, but to check whether the command starts with `cp` you need to write like this `command[0] == "cp"`